graphql/resolvers.go

```diff
@@ -831,6 +831,16 @@ func (r *mutationResolver) RestartTask(ctx context.Context, taskID string) (*res
 	return apiTask, err
 }
 
+func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription restModel.APISubscription) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	username := usr.Username()
+	err := r.sc.SaveSubscriptions(username, []restModel.APISubscription{subscription})
```

Comment:
I am in the middle of adding validation before this function is called to ensure the project, task, build or version exists